src/service/poc_mobile.proto

```diff
@@ -438,6 +438,90 @@ message radio_reward {
   repeated boosted_hex boosted_hexes = 10;
 }
 
+enum boosted_hex_status {
+  boosted_hex_status_eligible = 0;
+  boosted_hex_status_location_score_below_threshold = 1;
+  boosted_hex_status_radio_threshold_not_met = 2;
+  boosted_hex_status_service_provider_ban = 3;
+}
+
+message radio_reward_v2 {
+
+  message location_trust_score {
+    uint64 meters_to_asserted = 1;
+    // value is 0.0 to 1.0 multiplied by 1000
+    uint32 trust_score = 2;
+  }
+
+  message covered_hex {
+    uint64 location = 1;
+    // modeled coverage points * rank_multiplier * assignment_multiplier
+    uint64 base_coverage_points = 2;
+    // base_coverage_points * (boosted_multiplier - 1)
+    uint64 boosted_coverage_points = 3;
+
+    oracle_boosting_assignment urbanized = 4;
+    oracle_boosting_assignment footfall = 5;
+    oracle_boosting_assignment landtype = 6;
+    // value is 0.0 to 1.0 multiplied by 1000
+    uint32 assignment_multiplier = 7;
+    // rank for this hex, 1 indexed
+    uint32 rank = 8;
+    // value is 0.0 to 1.0 multiplied by 1000
+    uint32 rank_multiplier = 9;
+
+    // set to 0 is radio is ineligible for boosted rewards
+    // or hex is not boosted by service provider
+    uint32 boosted_multiplier = 10;
+  }
+
+  // Public key of the hotspot of the radio
+  bytes hotspot_key = 1;
+  // cbsd Id of the radio
+  string cbsd_id = 2;
+
+  // sum of base_coverage_points for all covered_hexes
+  uint64 base_coverage_points = 3;
+  // sum of boosted_coverage_points for all covered hexes
+  uint64 boosted_coverage_points = 4;
```

Comment:
Is `base_coverage_points` + `boosted_coverage_points` = Accumulated coverage points for the radio?  (`coverage_points` in `message radio_reward` ).

Is the same for `base_poc_reward` + `boosted_poc_reward` = radio's poc reward ?